pkg/scheduler/apis/config/types.go

```diff
@@ -74,8 +74,8 @@ type KubeSchedulerConfiguration struct {
 	// Example: if the cluster size is 500 nodes and the value of this flag is 30,
 	// then scheduler stops finding further feasible nodes once it finds 150 feasible ones.
 	// When the value is 0, default percentage (5%--50% based on the size of the cluster) of the
-	// nodes will be scored.
-	PercentageOfNodesToScore int32
+	// nodes will be scored. It is overridden by profile level PercentageOfNodesToScore.
+	PercentageOfNodesToScore *int32
```

Comment:
If we want to keep the API stable, why can we change this? Because no difference to end-users? 
So the conclusion is if we can keep the consistent experience to users, no breaking change, then we can make some modifications, right?

Reply:
Keep it consistent with the profile percentage. We need pointer to tell the difference between 0 and not set (nil) for profile percentage. If it's the latter, the global percentage will be used.